### PR TITLE
Added the blur function to the Image related classes

### PR DESCRIPTION
**Pull request by Seb-C**

### CODE REVIEW

**Review by tonyyb — APPROVED** [review]

File: classes/image/imagemagick.php [hunk 1]
```diff
@@ -158,6 +158,14 @@ protected function _mask($maskimage)
 		$this->exec('convert', $command);
 	}
 
+	protected function _blur($radius, $sigma)
+	{
+		extract(parent::_blur($radius, $sigma));
+
+		$image = '"'.$this->image_temp.'"';
+		$this->exec('convert', $image.' -blur '.$radius.'x'.$sigma.' '.$image);
```
Comment (Meroje) [inline classes/image/imagemagick.php:166]: `$radius.'x'.$sigma` should be quoted for shell use, or casted to actual floats somewhere (also be careful of the tostring behavior on floats)

Comment (tonyyb) [reply classes/image/imagemagick.php:166]: Agree .. But no function on this Class uses quotes or casted values.

Comment (Meroje) [reply classes/image/imagemagick.php:166]: All other parameters are expected to be integers, harm can be done with strings, but not by automatic toString format. At least use number_format to get spaces and commas right.

Comment (Seb-C) [reply classes/image/imagemagick.php:166]: Formatting to string is already handled in the queue system, like in others methods: https://github.com/novius-os/fuelphp-core/blob/1.7/master/classes/image/driver.php#L848

**Review by Meroje — APPROVED** [review]

**Review by shaoshiva — APPROVED** [review]

**Pull request merged**